### PR TITLE
Hotfix anonymous results

### PR DESCRIPTION
**Pull request by dave-mills**
In the main organisation dashboard, currently the code to hide "other institutions' results" when there are too few results to be anonymous is disabled. 

This PR re-enables that feature and fixes an issue with the checking. Now, When there are fewer than 10 projects and 3 separate organisations contributing to any filtered set of results, the results will be hidden for other institution's initiatives.

Screenshot of what should happen when filters are applied to return a results set that is too small: 
CleanShot 2024-03-18 at 10 55 04

### CODE REVIEW

File: resources/views/organisations/show.blade.php [hunk 1]
```diff
@@ -6,7 +6,7 @@
         @if(! $organisation->agreement_signed_at && $organisation->admins->contains(auth()->user()))
         <div class="alert alert-warning show text-dark my-3">Before continuing, please check and agree to the data sharing agreement, available on the Settings tab.</div>
         @elseif(! $organisation->agreement_signed_at && !$organisation->admins->contains(auth()->user()))
-        <div class="alert alert-warning text-dark my-3">This institution has not yet digitally signed the data sharing agreement. An institution admin should update the settings.</div>
+        <div class="alert alert-warning text-dark my-3">This institution has not yet agreed to the data sharing agreement. An institution admin should update the settings.</div>
```
Comment (dave-mills) [inline resources/views/organisations/show.blade.php:9]: This hotfix branch came from `main`, so this change is one that was made on main as a previous hotfix and not yet merged into dev. The "agreed to" text is correct.

**Comment by dan-tang-ssd** [comment]
Thank you Dave. 

I did below to update my local env:
```
git checkout dev
git pull
git checkout hotfix-anonymous-results
git pull
npm run build
npm run dev
```

---

The two changes in this PR are working well.

In my local env testing, there is one thing unexpected.

In dashboard, it shows message "There is no fully assessed initiative" while we have fully assessed initiatives.

image

**Comment by dave-mills** [comment]
Thanks Dan,

Is the unexpected message something related to this hotfix, do you think? Should we investigate that here, or push this hotfix and then investigate it separately?

**Comment by dan-tang-ssd** [comment]
I just did a checking in live env, found that it also shows the same message "There is no fully assessed initiative".
I have only one initiative, I have completed the assessment for red flags and principles.

image

image

**Comment by dan-tang-ssd** [comment]
> Thanks Dan,
> 
> Is the unexpected message something related to this hotfix, do you think? Should we investigate that here, or push this hotfix and then investigate it separately?

I think the unexpected message is not related to this hotfix. As it is also showed in live env...

Let's investigate it separately.

**Review by dan-tang-ssd — APPROVED** [review]

**Pull request merged**